fcrepo-import/src/main/java/org/fcrepo/importer/ArgParser.java

```diff
@@ -26,13 +27,19 @@
 import org.apache.commons.cli.ParseException;
 
 /**
+ * Command-line arguments parser.
+ *
  * @author awoods
+ * @author escowles
  * @since 2016-08-29
  */
 public class ArgParser implements org.fcrepo.importexport.ArgParser {
 
     final private Options cmdOptions;
 
+    public static final String DEFAULT_RDF_EXT = ".ttl";
```

Comment:
I'm having trouble resolving the circular dependencies with the ArgParser class — I thought it would go in the driver project, but the Import and Export tests need it to parse the options.  But then it can't go in Common because it instantiates the Importer and Exporter classes...

Maybe we're overthinking this — do we need four modules for seven classes?  Can we just combine them back into one package now?